src/coreclr/vm/threadsuspend.cpp

```diff
@@ -1997,15 +1997,6 @@ CONTEXT* AllocateOSContextHelper(BYTE** contextBuffer)
             pfnInitializeContext2(buffer, context, &pOSContext, &contextSize, xStateCompactionMask):
             InitializeContext(buffer, context, &pOSContext, &contextSize);
 
-        // if AVX is supported set the appropriate features mask in the context
```

Comment:
With this removed, the code above that sets the supportAVX based on the FeatureMask can be removed too.

Reply:
Right, even before `supportAVX` was redundant, since `SetXStateFeaturesMask` performs the same check - to filter out what is not supported.